tests/sqllogictests/suites/duckdb/join_small_block/semianti/antijoin.test

```diff
@@ -38,10 +38,8 @@ SELECT * FROM left_table ANTI JOIN (SELECT a as foo from right_table where b = 1
 43	1	1
 
 # Should throw error when filtering on column in right table
-statement error
+statement error 1065
 SELECT * FROM left_table ANTI JOIN right_table ON left_table.a = right_table.a WHERE right_table.a < 43;
-----
-Binder Error
```

Comment:
it seems the format doesn't make sense: https://github.com/risinglightdb/sqllogictest-rs/releases/tag/v0.18.0

If we don't remove `----`, what will happen?

Reply:
I think showing specific error info is better.

Reply:
> I think showing specific error info is better.

how about inline error? Regular expressions have not been supported in the multiline error yet.

Reply:
make sense